### ISSUE TITLE
New pre-made build?

### ISSUE THREAD

**Issue by felipecrs** [sickcodes/Docker-OSX#760]
Hi,

I see you just released support for Sonoma. Thanks a ton!

I wonder if you are considering building a new pre-made variant for Sonoma as well, since Catalina is quite old by now and several applications are no longer compatible with it.

**Comment by sickcodes** [comment]
Yeah I might make one this week. I wrote myself instructions last time, so I'll go ahead and create one.

**Comment by ebouther** [comment]
Any update ? I'd be interested in a pre-made variant too! Thanks!

**Comment by felipecrs** [comment]
As an alternative: https://github.com/dockur/macos

EDIT: Never mind, that project does not provide pre-made builds.
